pandas/core/frame.py

```diff
@@ -390,7 +391,13 @@ def _constructor_expanddim(self):
     # ----------------------------------------------------------------------
     # Constructors
 
-    def __init__(self, data=None, index=None, columns=None, dtype=None, copy=False):
+    def __init__(self,
+                 data: Optional[Union[np.ndarray, Iterable, Dict[str, AnyArrayLike],
+                                "DataFrame"]] = None,
```

Comment:
I have added Iterable to type of data, which will cause an, which I can't figure out why it's happening or how to fix it. Also added types of k,v of Dict(which might not be perfect). @WillAyd Some help.

Reply:
If this is a hang up let's just remove this particular annotation for now to keep this moving. Can address in folllow up